crates/api_models/src/tokenization.rs

```diff
@@ -22,8 +22,29 @@ pub struct GenericTokenizationResponse {
 pub struct GenericTokenizationRequest {
     /// Customer ID for which the tokenization is requested
     #[schema(value_type = String, example = "12345_cus_01926c58bc6e77c09e809964e72af8c8")]
-    pub customer_id: GlobalCustomerId,
+    pub customer_id: id_type::GlobalCustomerId,
     /// Request for tokenization which contains the data to be tokenized
     #[schema(value_type = Object,example = json!({ "city": "NY", "unit": "245" }))]
     pub token_request: masking::Secret<serde_json::Value>,
 }
+
+#[cfg(all(feature = "v2", feature = "tokenization_v2"))]
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
+pub struct DeleteTokenDataRequest {
+    /// Customer ID for which the tokenization is requested
+    #[schema(value_type = String, example = "12345_cus_01926c58bc6e77c09e809964e72af8c8")]
+    pub customer_id: id_type::GlobalCustomerId,
+    /// Request for tokenization which contains the data to be tokenized
+    #[schema(value_type = String, example = "12345_pms_01926c58bc6e77c09e809964e72af8c8")]
+    pub session_id: id_type::GlobalPaymentMethodSessionId,
+}
+
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
+pub struct DeleteTokenDataResponse {
+    /// Unique identifier returned by the tokenization service
+    #[schema(value_type = String, example = "12345_tok_01926c58bc6e77c09e809964e72af8c8")]
+    pub id: id_type::GlobalTokenId,
+    ///Whether the token data was deleted successfully
+    #[schema(example = false)]
+    pub token_data_deleted: bool,
```

Comment:
Why do we need this additional field? We can show successful deletion in the response status code